geo/src/main/java/org/odk/collect/geo/geopoly/GeoPolyActivity.java

```diff
@@ -274,7 +276,7 @@ public void initMap(MapFragment newMapFragment) {
         if (restoredPoints != null) {
             points = restoredPoints;
         }
```

Comment:
I've added some improvements in https://github.com/getodk/collect/pull/6083/commits/82da1e1a61ba7def075f452040c4e9141380e5a8 to avoid passing such default values but here it's java so it would require reordering the parameters or adding a secondary constructor. I would rather leave it as-is and wait to convert this class to Kotlin.